src/script/sign.h

```diff
@@ -73,6 +76,565 @@ struct SignatureData {
     void MergeSignatureData(SignatureData sigdata);
 };
 
+
+
+// Note: These constants are in reverse byte order because serialization uses LSB
+static constexpr uint32_t PSBT_MAGIC_BYTES = 0x74627370;
```

Comment:
You could also write it as `uint8_t PSBT_MAGIC_BYTES[4] = "PSBT";`. Byte arrays can be serialized directly now.

Reply:
Done